src/master-playlist-controller.js

```diff
@@ -258,6 +259,12 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     this.setupSegmentLoaderListeners_();
 
+    if (this.experimentalBufferBasedABR) {
+      this.startABRTimer_();
```

Comment:
Should we be starting the timer before the first `play`?

Reply:
we probably need to update this based on the preload setting. If preload is none or metedata (i.e. when we shouldn't have downloaded any segments) there's no point to it but with preload auto, I think we should.

I'll add updating this as part of the story to fix preload=none